c7n/policy.py

```diff
@@ -201,24 +201,26 @@ def run(self, *args, **kw):
             return
 
         with self.policy.ctx:
+            region        = (self.policy.options.region or 'default').ljust(14, ' ')
```

Comment:
the longest aws region string is 14 characters (any of the asia pacific regions)